Sources/swift-package-manager/SwiftPM.swift

```diff
@@ -22,14 +22,14 @@ let execName = (try? AbsolutePath(validating: firstArg).basenameWithoutExt) ??
 
 @main
 struct SwiftPM {
-    static func main() {
+    static func main() async {
```

Comment:
I'm not sure we can do this until the CI has been updated to macOS 12?

Reply:
You're right 😭

Reply:
main.swift here we come